test/acceptance/features/steps/steps.py

```diff
@@ -200,6 +200,10 @@ def sbr_is_applied(context):
         ns = None
     assert sbr.create(sbr_yaml, ns) is not None, "Service binding not created"
 
+    time.sleep(10)
+    json_path = "{.status.secret}"
+    context.sb_secret_name = sbr.get_servicebinding_info_by_jsonpath(context.sbr_name, context.namespace.name, json_path)
```

Comment:
We should read secret name in steps where it really needed.

Reply:
Made some changes here 7de50fb